### ISSUE TITLE
Google Contact groups do not appear on PC client/extension compose

### ISSUE THREAD

**Issue by sqlslinger** [klinker-apps/messenger-issues#840]
- [x] I have [searched](https://github.com/klinker-apps/messenger-issues/issues), by keyword, for my problem to verify that there aren't any open or closed issues around this behavior.
- [x] I have [searched the help page](https://messenger.klinkerapps.com/help) to verify that it does not have any topics relevant to this issue.
- [x] I have verified that this is something that is reproducible.

**Device with issue:** Win10 1803 64-bit 
**Primary device:** OnePlus 3T running Android 8.0 patch level 11/1/2018
**App version:** Windows v3.2.1, app 4.6.2.2382

**Describe the bug:**
Some contacts and many contact groups not appearing on Windows Client or Chrome extension

**Steps to reproduce:**

1. Go to Windows Client (or Chrome Extension)
2. Click on '+' to compose
3. Type first letters of group name
4. Don't see groups I expect.  See only a few contacts with those letters

**Expected behavior:**
I want to see the contact groups (labels from google contacts)

**Screenshots:**
Group I would like to see when I type Luth:
![image](https://user-images.githubusercontent.com/17749699/53274714-34df8880-36c6-11e9-9cb9-d51621249100.png)

What I actually see:
![image](https://user-images.githubusercontent.com/17749699/53274785-86881300-36c6-11e9-85c9-ad61c7909869.png)

**Comment by klinker24** [comment]
Have you tried just logging out and back in to the desktop app?

**Comment by sqlslinger** [comment]
Yes, I did try refresh on phone, log out on windows client, log back in, same test as above, same results.  

Thanks for the quick reply!

Vince

**Comment by klinker24** [comment]
Refreshing the account is not the same as resyncing your contacts. That will not do anything for this.

**Comment by evolver21** [comment]
I'm experiencing exactly the same issue. I have narrowed this issue down to the limit in contacts per label. As soon as I add more than 150 contacts into a label, the label disappears from the "Groups" tab in mobile app. Regardless, no matter what I do, I can't get any of the labels to appear in the Web App (tried all of the steps above). Thanks

**Comment by sqlslinger** [comment]
Luke, please respond to evolver21's comment.  I agree that this issue occurs with large groups, since I have over 150 contacts in a label also (but less than 300).

**Comment by klinker24** [comment]
If I had a solution, I would provide it. Sorry, I do not have an answer to this, at this time. You can create the group message from the phone app and respond to it from the web, if the web isn't displaying the group.

I do not limit the groups. It isn't as though I have set up a limit of 150 contacts to display.
